### DISCUSSION TITLE
Per-type graphs for code security insights on the organization-level Security tab

### DISCUSSION THREAD

**Discussion by ebickle** [community/community#74874] [Code Security]
### Select Topic Area

Product Feedback

### Body

Appreciate the new graphs and dashboard information on the organizational-level Security tab.

One thing I noticed is that the sheer volume of dependabot alerts (which can be noisy) tends to drown out the work we're doing in CodeQL and Secret Scanning. Would it be possible to get graphs by "category"? e.g. one for dependencies, another for code scanning, and another for secrets. Thanks!

**Comment by obvioussean** [comment]
Thanks for the feedback! We do have a `tool` filter:

<img width="400" alt="image" src="https://github.com/community/community/assets/15038374/4f125aa4-6fe9-4027-b4f8-7581b81d6bc4">


Would that address your feedback?

**Reply by ebickle** [reply]
That worked, thanks!

**Reply by obvioussean** [reply]
Great! Was the keyword `tool` intuitive to you or did you think it meant something else?

**Reply by ebickle** [reply]
I wasn't aware that it was there, so I don't think it's intuitive at first. If you think it's a common scenario for people to want to select a single tool, having some other UI next to the search box could be a help.

A bit like the "repositories" tab on a GitHub organization. Next to the "find" search box are drop downs for Type/Language/Sort. A drop down in a similar spot for "Tool" would be helpful. The main use case for us is that Dependabot alerts (lots of low regex ddos devDependencies!) tend to vastly overwhelm the smaller #s of alerts from CodeQL and secret scanning.  😁
